.env

```diff
@@ -1,13 +1,12 @@
 # Example .env file
 
-BRUPOP_CONTAINER_IMAGE=109276217309.dkr.ecr.us-west-2.amazonaws.com/brupop:latest
+BRUPOP_CONTAINER_IMAGE=public.ecr.aws/bottlerocket/bottlerocket-update-operator:v0.2.0
```

Comment:
Nit: Since currently 0.2.0 tag is not existing in brupop public ECR, should we finish this step once we publish brupop image to `public.ecr.aws/bottlerocket/bottlerocket-update-operator:v0.2.0`？ Maybe we can add one step in MCM like Pointing brupop resource yaml file to correct docker image?

Reply:
Another option could be that we don't merge `0.2.0` onto `develop` until the CM is complete. Does that sound OK?

Reply:
Yeah. that sounds good. : ) thanks.